src/Fable.React/Fable.React.fsproj

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Version>9.3.0</Version>
-    <PackageVersion>9.3.0</PackageVersion>
+    <Version>9.4.0</Version>
+    <PackageVersion>9.4.0</PackageVersion>
```

Comment:
Note for publish:

These lines should not be changed manually but instead a new entry should be made in the RELEASE_NOTES.md.

The build process will take care of updating the fsproj.

Reply:
fixed